src/Evolution/Imex/Protocols/ImexSystem.hpp

```diff
@@ -17,6 +17,8 @@ namespace imex::protocols {
 ///
 /// For efficiency, the tensors in each sector are required to be
```

Comment:
Can you expand this line? e.g. "the list of tags in the `tensors` type alias in each sector are required to be ..""